src/htmlminifier.js

```diff
@@ -182,7 +182,14 @@ function isExecutableScript(tag, attrs) {
 }
 
 function isStyleLinkTypeAttribute(attrValue) {
-  attrValue = trimWhitespace(attrValue).toLowerCase();
+  attrValue = trimWhitespace(attrValue);
+
+  if (typeof attrValue !== 'string') {
```

Comment:
Wouldn't be `typeof attrValue !== 'undefined'´be better?
Because the quotes are often optional (unquoted attribute values) so this would change `val=integer`to `val=''`.

Reply:
https://dev.w3.org/html5/html-author/Overview.src.html

> There are four slightly different syntaxes that may be used for attributes in HTML: empty, unquoted, single-quoted and double-quoted. All four syntaxes may be used in the HTML syntax, depending on what is needed for each specific attribute. However, in the XHTML syntax, attribute values must always be quoted using either single or double quotes.

https://mathiasbynens.be/notes/unquoted-attribute-values

Only undefined / empty values should be treated this way.

Reply:
I've made the update to only handle the 'undefined' values here.